src/bitops.c

```diff
@@ -556,6 +556,7 @@ int getBitOffsetFromArgument(client *c, robj *o, uint64_t *offset, int hash, int
  * On error C_ERR is returned and an error is sent to the client. */
 int getBitfieldTypeFromArgument(client *c, robj *o, int *sign, int *bits) {
     char *p = o->ptr;
+    size_t plen = sdslen(p);
```

Comment:
Minor: `p` is not declared as `sds`. Use o->ptr directly instead, or declare `p` as sds?

```suggestion
    sds p = o->ptr;
    size_t plen = sdslen(p);
```

Reply:
This is aligned with `getBitOffsetFromArgument` method implementation, so there is no special replacement.  
```
int getBitOffsetFromArgument(client *c, robj *o, uint64_t *offset, int hash, int bits) {
    long long loffset;
    char *err = "bit offset is not an integer or out of range";
    char *p = o->ptr;
    size_t plen = sdslen(p);
```
I tend to define `p` as sds.  So I modified the two logics together.